pkg/machines/components/vm/disks/diskAdd.jsx

```diff
@@ -439,16 +436,15 @@ export class AddDiskModalBody extends React.Component {
                 }
 
                 if (newBus)
-                    this.onValueChanged("busType", newBus);
+                    return ({ busType: newBus });
```

Comment:
Leaving out the onValueChanged can cause the `target` property to not be correct. When busType is changed through `onValueChanged`, it also triggers a change of `target` (e.g. from vda to sda). If this is omitted, a disk with a devide type of "sata" can end up with a target "vdb" instead of "sdb", which would be wrong.

Reply:
Alternatively you can change `target` here together with busType.

Reply:
Got it! I will fix, thanks!